pulp_ansible/tests/functional/api/collection/v3/test_sync.py

```diff
@@ -203,6 +203,54 @@ def test_sync_with_missing_collection(self):
         msg = "absent.not_present does not exist"
         self.assertIn(msg, task_result["error"]["description"], task_result["error"]["description"])
 
+    def test_sync_with_proxy_auth(self):
```

Comment:
This is really great. We needed some kind of test, any kind of test and you did that.

The challenge we'll have with this test is that it can't run locally without the CI deploying the proxy container. To solve that I propose we start and stop proxy.py (same code) via a pulp-smash pytest fixture. What do you think about that? Is that something you'd be willing to do? If not, maybe I can if that's helpful. I'm hoping we can switch to that style right away before tests like this get copied to more places.